packages/analytics/src/analytics.ts

```diff
@@ -75,8 +77,9 @@ export class Analytics<T extends AnalyticsEvent> {
         }
 
         const qs = encodeDataToQueryString(
-            this.instanceId,
-            this.sessionId,
+            // Random ID is better than constant because it helps clean the data in case it is accidentally logged multiple times.
```

Comment:
Please confirm with analytics guys that they do not do some analytics based on distinguish ids. 

How about having same id for all anonymized events from one user, does it make sense?